### ISSUE TITLE
Not able to add multiple cognitive search index in Azure AI studio

### ISSUE THREAD

**Issue by roshithrajan** [microsoft/sample-app-aoai-chatGPT#233]
I am not able to add multiple cognitive search index in Azure AI studio, i have two search index one is from storage account and other one from sharepoint . do we need to configure this using API or is this a limitation of the product 

![image](https://github.com/microsoft/sample-app-aoai-chatGPT/assets/5292177/683494d0-1162-4a29-9bc0-846782573b6d)

**Comment by sarah-widder** [comment]
Hi @roshithrajan currently only one index at a time can be used in both the UI and the API.

**Comment by peterjeitschko** [comment]
@sarah-widder I have several customer requests that also use multiple cognitive search indizes. Would be great if you could implement this. This is a pretty common request in my POV.

**Comment by roshithrajan** [comment]
Yes , it will be good, I have seen another issue is if we select the our own data still openai fetching data externally

**Comment by superpoussin22** [comment]
+1 for index choice in the UI
